qui/updater/style.py

```diff
@@ -0,0 +1,41 @@
+# coding=utf-8
+#
+# The Qubes OS Project, http://www.qubes-os.org
+#
+# Copyright (C) 2023  Piotr Bartman <prbartman@invisiblethingslab.com>
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301,
+# USA.
+import gi
+
+gi.require_version('Gtk', '3.0')  # isort:skip
+from gi.repository import Gtk, Gdk
+
+
+def load_css():
+    style_provider = Gtk.CssProvider()
+    css = b'''
```

Comment:
why not put this in a CSS file? I don't like mixing python with css here, makes it a bit harder to find when to change styles later.